app/main.py

```diff
@@ -1,2 +1,71 @@
+from typing import Hashable, Any
+from dataclasses import dataclass
+
+INITIAL_CAPACITY = 8
+
+
+@dataclass
+class Node:
+    key: Hashable
+    dict_hash: int
+    value: Any
+
+
 class Dictionary:
-    pass
+    def __init__(self, capacity: int = INITIAL_CAPACITY) -> None:
+        self.capacity = capacity
+        self.length = 0
+        self.threshold = 2 / 3
```

Comment:
The threshold calculation `self.threshold = 2 / 3` is incorrect. It should be `self.threshold = 2 / 3 * self.capacity` to represent the load factor threshold for resizing the hash table.